styles/style.css

```diff
@@ -37,6 +37,24 @@ body {
 	border-radius: 0.15em;
 }
 
+.btn-primary {
+		background-color: #ce5f31;
+}
```

Comment:
Watch out when using Bootstrap's ready-made buttons - they come with default styling such as the blue border, which you may want to overwrite here.